src/Files.App/ViewModels/FolderSettingsViewModel.cs

```diff
@@ -287,47 +287,59 @@ public ColumnsViewModel ColumnsViewModel
 		private static LayoutPreferences GetLayoutPreferencesForPath(string folderPath)
 		{
 			IUserSettingsService userSettingsService = Ioc.Default.GetRequiredService<IUserSettingsService>();
-			if (userSettingsService.FoldersSettingsService.EnableOverridingFolderPreferences)
+
+			var layout = LayoutPreferences.DefaultLayoutPreferences;
+
+			if (userSettingsService.FoldersSettingsService.EnableOverridingFolderPreferences || userSettingsService.LayoutSettingsService.EnableOverridingSortingPreferences)
```

Comment:
You could also reverse the condition of the `if` to avoid a nesting level with an earlier `return layout` after it.